src/Npgsql/NpgsqlDataReader.cs

```diff
@@ -973,6 +973,19 @@ internal async Task Cleanup(bool async, bool connectionClosing=false)
         /// <returns>The value of the specified column.</returns>
         public override DateTime GetDateTime(int ordinal) => GetFieldValue<DateTime>(ordinal);
 
+        /// <summary>
+        /// Gets the value of the specified column as a TimeSpan,
+        /// </summary>
+        /// <remarks>
+        /// PostgreSQL's interval type has has a resolution of 1 microsecond and ranges from
+        /// -178000000 to 178000000 years, while .NET's TimeSpan has a resolution of 100 nanoseconds
+        /// and ranges from roughly -29247 to 29247 years.
+        /// See http://www.postgresql.org/docs/current/static/datatype-datetime.html
+        /// </remarks>
+        /// <param name="ordinal">The zero-based column ordinal.</param>
+        /// <returns>The value of the specified column.</returns>
+        public TimeSpan GetTimeSpan(int ordinal) => GetFieldValue<TimeSpan>(ordinal);
+
```

Comment:
I would like to avoid such useless methods since they all fall into `GetFieldValue<T>`. It existed previously, but there are no good reasons to keep it alive.

Reply:
Yeah, all those methods are pretty useless nowadays, as we have `GetFieldValue<T>`.

Personally I don't care a lot, but I'd rather remove all of them or keep this one (which I only moved a few lines upwards, where it belongs IMO).

Reply:
As the method is already there, I wouldn't remove it and create needless breakage.